[2025-04-05T10:45:16.558+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-05T10:45:16.618+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T10:45:16.643+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T10:45:16.644+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-05T10:45:16.674+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_to_minio> on 2025-04-04 00:00:00+00:00
[2025-04-05T10:45:16.685+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=296) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-05T10:45:16.685+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'elt_pipeline', 'upload_to_minio', 'scheduled__2025-04-04T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/elt_pipeline.py', '--cfg-path', '/tmp/tmp09dj8zqp']
[2025-04-05T10:45:16.687+0000] {standard_task_runner.py:63} INFO - Started process 319 to run task
[2025-04-05T10:45:16.687+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask upload_to_minio
[2025-04-05T10:45:16.712+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-05T10:45:16.765+0000] {task_command.py:426} INFO - Running <TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [running]> on host d56545bf874e
[2025-04-05T10:45:16.901+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt_pipeline' AIRFLOW_CTX_TASK_ID='upload_to_minio' AIRFLOW_CTX_EXECUTION_DATE='2025-04-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-04T00:00:00+00:00'
[2025-04-05T10:45:16.903+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-05T10:45:16.920+0000] {logging_mixin.py:188} INFO - Error uploading file to Minio: [Errno 2] No such file or directory: '/opt/***/bitcoin-2021-01-01.csv'
[2025-04-05T10:45:16.920+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-05T10:45:16.922+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-05T10:45:16.944+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=elt_pipeline, task_id=upload_to_minio, run_id=scheduled__2025-04-04T00:00:00+00:00, execution_date=20250404T000000, start_date=20250405T104516, end_date=20250405T104516
[2025-04-05T10:45:16.986+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-05T10:45:17.017+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-05T10:45:17.022+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-04-05T11:10:26.511+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-05T11:10:26.554+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T11:10:26.564+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T11:10:26.565+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-05T11:10:26.583+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_to_minio> on 2025-04-04 00:00:00+00:00
[2025-04-05T11:10:26.590+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=357) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-05T11:10:26.591+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'elt_pipeline', 'upload_to_minio', 'scheduled__2025-04-04T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/elt_pipeline.py', '--cfg-path', '/tmp/tmp3yukdlnv']
[2025-04-05T11:10:26.592+0000] {standard_task_runner.py:63} INFO - Started process 380 to run task
[2025-04-05T11:10:26.592+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask upload_to_minio
[2025-04-05T11:10:26.614+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-05T11:10:26.665+0000] {task_command.py:426} INFO - Running <TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [running]> on host 1b61a216da47
[2025-04-05T11:10:26.769+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt_pipeline' AIRFLOW_CTX_TASK_ID='upload_to_minio' AIRFLOW_CTX_EXECUTION_DATE='2025-04-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-04T00:00:00+00:00'
[2025-04-05T11:10:26.771+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-05T11:10:26.783+0000] {logging_mixin.py:188} INFO - Error uploading file to Minio: [Errno 2] No such file or directory: '/opt/***/bitcoin-2021-01-01.csv'
[2025-04-05T11:10:26.784+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-05T11:10:26.785+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-05T11:10:26.800+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=elt_pipeline, task_id=upload_to_minio, run_id=scheduled__2025-04-04T00:00:00+00:00, execution_date=20250404T000000, start_date=20250405T111026, end_date=20250405T111026
[2025-04-05T11:10:26.853+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-05T11:10:26.877+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-05T11:10:26.881+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-04-05T15:55:35.088+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-05T15:55:35.126+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T15:55:35.134+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T15:55:35.134+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-05T15:55:35.146+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_to_minio> on 2025-04-04 00:00:00+00:00
[2025-04-05T15:55:35.154+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=537) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-05T15:55:35.154+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'elt_pipeline', 'upload_to_minio', 'scheduled__2025-04-04T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/elt_pipeline.py', '--cfg-path', '/tmp/tmp3nm2dsx9']
[2025-04-05T15:55:35.155+0000] {standard_task_runner.py:63} INFO - Started process 560 to run task
[2025-04-05T15:55:35.156+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask upload_to_minio
[2025-04-05T15:55:35.175+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-05T15:55:35.219+0000] {task_command.py:426} INFO - Running <TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [running]> on host a4017603cc0b
[2025-04-05T15:55:35.311+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt_pipeline' AIRFLOW_CTX_TASK_ID='upload_to_minio' AIRFLOW_CTX_EXECUTION_DATE='2025-04-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-04T00:00:00+00:00'
[2025-04-05T15:55:35.312+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-05T15:55:35.327+0000] {logging_mixin.py:188} INFO - Error uploading file to Minio: [Errno 2] No such file or directory: '/opt/***/bitcoin-2021-01-01.csv'
[2025-04-05T15:55:35.327+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-05T15:55:35.329+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-05T15:55:35.342+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=elt_pipeline, task_id=upload_to_minio, run_id=scheduled__2025-04-04T00:00:00+00:00, execution_date=20250404T000000, start_date=20250405T155535, end_date=20250405T155535
[2025-04-05T15:55:35.371+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-05T15:55:35.401+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-05T15:55:35.405+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-04-05T16:31:55.378+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-05T16:31:55.416+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T16:31:55.425+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T16:31:55.426+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-05T16:31:55.441+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_to_minio> on 2025-04-04 00:00:00+00:00
[2025-04-05T16:31:55.449+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'elt_pipeline', 'upload_to_minio', 'scheduled__2025-04-04T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/elt_pipeline.py', '--cfg-path', '/tmp/tmpj6y72jt3']
[2025-04-05T16:31:55.450+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=292) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-05T16:31:55.451+0000] {standard_task_runner.py:91} INFO - Job 7: Subtask upload_to_minio
[2025-04-05T16:31:55.451+0000] {standard_task_runner.py:63} INFO - Started process 316 to run task
[2025-04-05T16:31:55.469+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-05T16:31:55.516+0000] {task_command.py:426} INFO - Running <TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [running]> on host d060e19f414c
[2025-04-05T16:31:55.626+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt_pipeline' AIRFLOW_CTX_TASK_ID='upload_to_minio' AIRFLOW_CTX_EXECUTION_DATE='2025-04-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-04T00:00:00+00:00'
[2025-04-05T16:31:55.631+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-05T16:31:55.653+0000] {logging_mixin.py:188} INFO - Error uploading file to Minio: [Errno 2] No such file or directory: '/opt/***/bitcoin-2021-01-01.csv'
[2025-04-05T16:31:55.654+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-05T16:31:55.657+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-05T16:31:55.669+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=elt_pipeline, task_id=upload_to_minio, run_id=scheduled__2025-04-04T00:00:00+00:00, execution_date=20250404T000000, start_date=20250405T163155, end_date=20250405T163155
[2025-04-05T16:31:55.708+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-05T16:31:55.732+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-05T16:31:55.734+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-04-05T16:41:16.551+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-05T16:41:16.588+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T16:41:16.596+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T16:41:16.597+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-05T16:41:16.613+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_to_minio> on 2025-04-04 00:00:00+00:00
[2025-04-05T16:41:16.621+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=265) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-05T16:41:16.621+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'elt_pipeline', 'upload_to_minio', 'scheduled__2025-04-04T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/elt_pipeline.py', '--cfg-path', '/tmp/tmp6cvftmzl']
[2025-04-05T16:41:16.623+0000] {standard_task_runner.py:63} INFO - Started process 277 to run task
[2025-04-05T16:41:16.623+0000] {standard_task_runner.py:91} INFO - Job 4: Subtask upload_to_minio
[2025-04-05T16:41:16.643+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-05T16:41:16.688+0000] {task_command.py:426} INFO - Running <TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [running]> on host 658edd075563
[2025-04-05T16:41:16.796+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt_pipeline' AIRFLOW_CTX_TASK_ID='upload_to_minio' AIRFLOW_CTX_EXECUTION_DATE='2025-04-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-04T00:00:00+00:00'
[2025-04-05T16:41:16.797+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-05T16:41:16.811+0000] {logging_mixin.py:188} INFO - Error uploading file to Minio: [Errno 2] No such file or directory: '/opt/***/bitcoin-2021-01-01.csv'
[2025-04-05T16:41:16.812+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-05T16:41:16.813+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-05T16:41:16.825+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=elt_pipeline, task_id=upload_to_minio, run_id=scheduled__2025-04-04T00:00:00+00:00, execution_date=20250404T000000, start_date=20250405T164116, end_date=20250405T164116
[2025-04-05T16:41:16.890+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-05T16:41:16.921+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-05T16:41:16.926+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-04-05T16:43:44.837+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-05T16:43:44.874+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T16:43:44.881+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T16:43:44.882+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-05T16:43:44.900+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_to_minio> on 2025-04-04 00:00:00+00:00
[2025-04-05T16:43:44.909+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'elt_pipeline', 'upload_to_minio', 'scheduled__2025-04-04T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/elt_pipeline.py', '--cfg-path', '/tmp/tmpms4szfbb']
[2025-04-05T16:43:44.910+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=265) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-05T16:43:44.911+0000] {standard_task_runner.py:91} INFO - Job 4: Subtask upload_to_minio
[2025-04-05T16:43:44.912+0000] {standard_task_runner.py:63} INFO - Started process 277 to run task
[2025-04-05T16:43:44.932+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-05T16:43:44.977+0000] {task_command.py:426} INFO - Running <TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [running]> on host 1f8025531576
[2025-04-05T16:43:45.075+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt_pipeline' AIRFLOW_CTX_TASK_ID='upload_to_minio' AIRFLOW_CTX_EXECUTION_DATE='2025-04-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-04T00:00:00+00:00'
[2025-04-05T16:43:45.076+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-05T16:43:45.092+0000] {logging_mixin.py:188} INFO - Error uploading file to Minio: [Errno 2] No such file or directory: '/opt/***/bitcoin-2021-01-01.csv'
[2025-04-05T16:43:45.093+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-05T16:43:45.094+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-05T16:43:45.106+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=elt_pipeline, task_id=upload_to_minio, run_id=scheduled__2025-04-04T00:00:00+00:00, execution_date=20250404T000000, start_date=20250405T164344, end_date=20250405T164345
[2025-04-05T16:43:45.130+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-05T16:43:45.150+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-05T16:43:45.154+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-04-05T17:21:51.889+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-05T17:21:51.927+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T17:21:51.934+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T17:21:51.935+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-05T17:21:51.949+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_to_minio> on 2025-04-04 00:00:00+00:00
[2025-04-05T17:21:51.958+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=315) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-05T17:21:51.958+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'elt_pipeline', 'upload_to_minio', 'scheduled__2025-04-04T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/elt_pipeline.py', '--cfg-path', '/tmp/tmpsxhugvya']
[2025-04-05T17:21:51.960+0000] {standard_task_runner.py:63} INFO - Started process 327 to run task
[2025-04-05T17:21:51.960+0000] {standard_task_runner.py:91} INFO - Job 4: Subtask upload_to_minio
[2025-04-05T17:21:51.982+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-05T17:21:52.029+0000] {task_command.py:426} INFO - Running <TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [running]> on host 3662cb577b20
[2025-04-05T17:21:52.123+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt_pipeline' AIRFLOW_CTX_TASK_ID='upload_to_minio' AIRFLOW_CTX_EXECUTION_DATE='2025-04-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-04T00:00:00+00:00'
[2025-04-05T17:21:52.125+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-05T17:21:52.137+0000] {logging_mixin.py:188} INFO - Error uploading file to Minio: [Errno 2] No such file or directory: '/opt/***/bitcoin-2021-01-01.csv'
[2025-04-05T17:21:52.137+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-05T17:21:52.140+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-05T17:21:52.152+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=elt_pipeline, task_id=upload_to_minio, run_id=scheduled__2025-04-04T00:00:00+00:00, execution_date=20250404T000000, start_date=20250405T172151, end_date=20250405T172152
[2025-04-05T17:21:52.177+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-05T17:21:52.202+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-05T17:21:52.205+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-04-05T17:29:08.119+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-05T17:29:08.153+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T17:29:08.160+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T17:29:08.160+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-05T17:29:08.173+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_to_minio> on 2025-04-04 00:00:00+00:00
[2025-04-05T17:29:08.180+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=307) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-05T17:29:08.181+0000] {standard_task_runner.py:63} INFO - Started process 319 to run task
[2025-04-05T17:29:08.181+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'elt_pipeline', 'upload_to_minio', 'scheduled__2025-04-04T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/elt_pipeline.py', '--cfg-path', '/tmp/tmpwsmrpu4z']
[2025-04-05T17:29:08.182+0000] {standard_task_runner.py:91} INFO - Job 4: Subtask upload_to_minio
[2025-04-05T17:29:08.199+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-05T17:29:08.238+0000] {task_command.py:426} INFO - Running <TaskInstance: elt_pipeline.upload_to_minio scheduled__2025-04-04T00:00:00+00:00 [running]> on host de3d8b486e6c
[2025-04-05T17:29:08.328+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt_pipeline' AIRFLOW_CTX_TASK_ID='upload_to_minio' AIRFLOW_CTX_EXECUTION_DATE='2025-04-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-04T00:00:00+00:00'
[2025-04-05T17:29:08.329+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-05T17:29:08.343+0000] {logging_mixin.py:188} INFO - Error uploading file to Minio: [Errno 2] No such file or directory: '/opt/***/bitcoin-2021-01-01.csv'
[2025-04-05T17:29:08.344+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-05T17:29:08.345+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-05T17:29:08.356+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=elt_pipeline, task_id=upload_to_minio, run_id=scheduled__2025-04-04T00:00:00+00:00, execution_date=20250404T000000, start_date=20250405T172908, end_date=20250405T172908
[2025-04-05T17:29:08.399+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-05T17:29:08.418+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-05T17:29:08.421+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
